common/static/common/templates/discussion/thread.underscore

```diff
@@ -16,8 +16,19 @@
             <% } %>
             <div class="post-btn-count-wrapper">
                 <ul class="post-info-list">
-                    <li>By <strong>Babar Hameed</strong></li>
-                    <li>4 days ago</li>
+                    <li>By <strong>
+                    <% if (typeof username !== 'undefined') { %>
+                    <a href="<%- user_url %>" class="username"><%- username %></a>
```

Comment:
Just realized that `username` needs to be changed here. Colaraz is using `first_name + second name`.